content/en/blog/2021/istio-community-meetup-china/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "The first local Istio meetup is here!"
+description: "Join Istio community experts in China."
```

Comment:
```suggestion
title: "Join us at the Istio Community Meetup in China"
description: "The Chinese Istio community comes together in Beijing."
```
Lots of places have run Istio meetups before; suggesting a more specific heading. (Not sure about the description yet)

(Will this be more useful when put on the ZH blog?)